sites/public/lib/filterUrlLink.tsx

```diff
@@ -0,0 +1,31 @@
+import { encodeToFrontendFilterString, ListingFilterState } from "@bloom-housing/ui-components"
+
+export function getFilterUrlLink(eligibilityRequirements) {
+  /** This is where any logic for the filtered results from the eligibility questionnaire should be located.
+   *  Example if household size dictated bedrooms:
+   *  switch (eligibilityRequirements.householdSizeCount.toString()) {
+   *  case "one":
+   *  case "two":
+   *    params.bedrooms = 1
+   *    break
+   *  case "three":
+   *    params.bedrooms = 2
+   *    break
+   *  case "four":
+   *    params.bedrooms = 3
+   *    break
+   *  default:
+   *    params.bedrooms = 4
+   *  }
+   */
```

Comment:
In general I'm in favor of verbose comments like this one, but I think I'm going to recommend removing this :) Reasoning: this comment specifically mentions the eligibility questionnaire, but this `getFilterUrlLink` is more general-purpose (ie could be used in the future by other parts of the frontend). And there's something about having code in a comment that feels like it's saying "we should do this, but it's blocked by X" which I don't think is the case.